templates/voting.html

```diff
@@ -6,5 +6,28 @@
 {% endblock stylesheet %}
 
 {% block body %}
-<h1>Exists.</h1>
-{% endblock %}
+
+<nav>
+    <button><a href="/">Home</a></button>
+    <button><a href="/voting">Voting</a></button>
+    <button><a href="/about">About Us</a></button>
+</nav>
+
+<nav>
+    <ul class="backwards"><a href="/voting/previous">← Previous Week's Poll</a></li>
+    <ul class="forwards"><a href="/voting/next">Preview Next Week's Poll →</a></li>
+</nav>
+<h1 style="color:green">Cast your Votes!</h1>
+<h2>Is the Battleship in Wilmington, North Carolina really haunted?</h2>
```

Comment:
This will all be changed later to take data from the server, but it's good for now as proof of concept